Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Instance/Instance.cpp

```diff
@@ -542,9 +542,9 @@ namespace AzToolsFramework
             return aliasPathResult;
         }
 
-        EntityAlias Instance::GenerateEntityAlias()
-        {
-            return AZStd::string::format("Entity_%s", AZ::Entity::MakeId().ToString().c_str());
+        EntityAlias Instance::GenerateEntityAlias(AZStd::string_view e_name)
+        { 
+            return AZStd::string::format("%s-%s", e_name.data(), AZ::Entity::MakeId().ToString().c_str());
         }
```

Comment:
Instead of abbreviating, spell out the entire variable name, entityName. That also matches the declaration.